accounts/abi/bind/template.go

```diff
@@ -26,13 +30,15 @@ type tmplData struct {
 
 // tmplContract contains the data needed to generate an individual contract binding.
 type tmplContract struct {
-	Type        string                 // Type name of the main contract binding
-	InputABI    string                 // JSON ABI used as the input to generate the binding from
-	InputBin    string                 // Optional EVM bytecode used to denetare deploy code from
-	Constructor abi.Method             // Contract constructor for deploy parametrization
-	Calls       map[string]*tmplMethod // Contract calls that only read state data
-	Transacts   map[string]*tmplMethod // Contract calls that write state data
-	Events      map[string]*tmplEvent  // Contract events accessors
+	Type          string                 // Type name of the main contract binding
+	InputABI      string                 // JSON ABI used as the input to generate the binding from
+	InputBin      string                 // Optional EVM bytecode used to denetare deploy code from
+	Constructor   abi.Method             // Contract constructor for deploy parametrization
+	Calls         map[string]*tmplMethod // Contract calls that only read state data
+	Transacts     map[string]*tmplMethod // Contract calls that write state data
+	Events        map[string]*tmplEvent  // Contract events accessors
+	Structs       map[string]*tmplStruct // Contract struct type definitions
+	SortedStructs sortedStructs          // Sorted struct type definitions by struct name.
```

Comment:
Because we can't get the raw struct name from abi. So I have to assign these structs a name like `Struct0, Struct1`. It just looks tidy when I sort them by name.